src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 20px 8px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+.search__bar-big {
+  position: relative;
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-bottom: 20px;
+}
```

Comment:
You have lots of repetitive styles between `search__bar-big` and `search__bar-small`.
Add common class, like `search__bar` for these two elements and move similar styles there.

Same for `search__bar-input`